include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -160,31 +160,13 @@ SECTIONS
  * zephyr_linker_sources() Cmake function.
  */
 #include <snippets-text-start.ld>
-#if defined(CONFIG_SW_VECTOR_RELAY)
-	KEEP(*(.vector_relay_table))
-	KEEP(*(".vector_relay_table.*"))
-	KEEP(*(.vector_relay_handler))
-	KEEP(*(".vector_relay_handler.*"))
-#endif
-
-	_vector_start = .;
-	KEEP(*(.exc_vector_table))
-	KEEP(*(".exc_vector_table.*"))
-
-	KEEP(*(IRQ_VECTOR_TABLE))
-
-	KEEP(*(.vectors))
-
-	KEEP(*(.openocd_dbg))
-	KEEP(*(".openocd_dbg.*"))
 
 #ifdef CONFIG_KINETIS_FLASH_CONFIG
 	. = CONFIG_KINETIS_FLASH_CONFIG_OFFSET;
 	KEEP(*(.kinetis_flash_config))
 	KEEP(*(".kinetis_flash_config.*"))
 #endif
 
-	_vector_end = .;
```

Comment:
Do we know if kinetis_flash_config is a part of the vector relay table? My intuition says no. So in my mind, this is correct and shouldn't even have been inside _vector_end, to begin with. However, I don't know the architecture so this could be wrong.

Reply:
Yes, the new design will keep it outside, but someone should should shout if that is wrong.

Reply:
No, it's not part of the vector relay table and shouldn't have been inside _vector_end. openocd_dbg shouldn't have been inside either.

Reply:
Thanks for confirming. Where should openocd_dbg go? Does it have any strict placement restrictions?